GoogleChromeLabs/ps-analysis-tool
# Feature: High level aggregated CSV download for sitemaps.

## Description
This PR aims to allow users to download aggregated CSV data when analysing a sitemap.

## Relevant Technical Choices
- A new function to generate aggregated CSV has been introduced to calculate the aggregated data of the sitemap.

## Testing Instructions
- Clone this branch
- In the terminal run `npm run build:cli:dashboard`.
- Then run analysis on a sitemap.
- After analysis is done, open the report and download the report from sitemap landing page.
- You should see a `report.csv` in the zip file, which will contain the data of the landing page.

## Additional Information:



## Screenshot/Screencast



---

## Checklist






Fixes #809
